### ISSUE TITLE
Landscape | only partial file shown

### ISSUE THREAD

**Issue by boskokg** [rbcprolabs/packages.flutter#54]
**Describe the bug**
When I open pdf in the landscape mode, only partial of the file is loaded.

**Expected behavior**
User should be able to view the entire page (probably by scroll up/down)

**Smartphone (please complete the following information):**
 - Device: Android 10

**Additional context**
Is there a way to workaround this problem? I used the provided example

**Comment by SergeShkurko** [comment]
I'll try to fix it soon

**Comment by SergeShkurko** [comment]
as hotfix you can usage it code:
```dart
PDFView.builder(
  controller: pageController,
  document: snapshot.data,
  builder: (PDFPageImage pageImage, bool isCurrentIndex) {
    Widget image = ExtendedImage.memory(
      pageImage.bytes,
      fit: BoxFit.fitHeight, // <- set fitHeight instead fitWidth
      mode: ExtendedImageMode.gesture,
      initGestureConfigHandler: (_) => GestureConfig(
        minScale: 1,
        animationMinScale: .75,
        maxScale: 2,
        animationMaxScale: 2.5,
        speed: 1,
        inertialSpeed: 100,
        inPageView: true,
        initialScale: 1.0,
        cacheGesture: false,
      ),
      onDoubleTap: (ExtendedImageGestureState state) {
        final pointerDownPosition = state.pointerDownPosition;
        final begin = state.gestureDetails.totalScale;
        double end;

        if (begin == _doubleTapScales[0]) {
          end = _doubleTapScales[1];
        } else {
          end = _doubleTapScales[0];
        }

        state.handleDoubleTap(
          scale: end,
          doubleTapPosition: pointerDownPosition,
        );
      },
    );
    if (isCurrentIndex) {
      image = Hero(
        tag: 'pdf_view' + pageImage.pageNumber.toString(),
        child: image,
      );
    }
    return image;
  },
);
```

Today I will prepare an update

**Comment by boskokg** [comment]
@SergeShkurko thank you

**Comment by SergeShkurko** [comment]
fixed in [3.0.0](https://pub.dev/packages/native_pdf_view/versions/3.0.0)
